### PR TITLE
Update opentelemetry-specification version to v1.39.0

### PR DESCRIPTION
**Pull request by opentelemetrybot**
Update opentelemetry-specification version to `v1.39.0`.

See https://github.com/open-telemetry/opentelemetry-specification/releases/tag/v1.39.0.

### CODE REVIEW

**Comment by chalin** [comment]
@trask @reyang @open-telemetry/specs-logs-approvers - please review and approve the doc changes I've added to this PR.

**Review by trask — COMMENTED** [review]
I'd suggest leaving the "Logs Bridge API" language in place, since that's the only thing that's stable, and waiting a bit to update the language to reflect the more general (desired future state) "Logs API"

**Comment by trask** [comment]
cc @open-telemetry/semconv-event-approvers

**Comment by chalin** [comment]
@trask et all: Might we want to get the updated spec published before KubeCon? Can we get these changes in and revisit later if necessary?

File: content/en/docs/specs/status.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ same as the **Protocol** status.
 
 ### [Logging][]
 
-- {{% spec_status "Bridge API" "otel/logs/bridge-api" "Status" %}}
+- {{% spec_status "API" "otel/logs/api" "Status" %}}
```
Comment (trask) [inline content/en/docs/specs/status.md:95]: while there is a new concept in the spec called "Logs API", it's not stable, only the previous concept "Logs Bridge API" is stable

File: content/en/docs/zero-code/python/logs-example.md [hunk 2]
```diff
@@ -20,11 +20,11 @@ Another way this is accomplished is through Python's support for
 auto-instrumentation of logs. The example below is based on the logs example in
 [OpenTelemetry Python repository][].
 
-> There is a logs bridge API; however, it is different from the Traces and
-> Metrics API, because it's not used by application developers to create logs.
-> Instead, they would use this bridge API to setup log appenders in the standard
-> language-specific logging libraries. More information can be found
-> [here](/docs/specs/otel/logs/bridge-api).
+> There is a logs API, however it is different from the Traces and Metrics API,
+> because it's not used by application developers to create logs. Instead, they
```
Comment (trask) [inline content/en/docs/zero-code/python/logs-example.md:24]: > it's not used by application developers to create logs

this is not true of the new (experimental) "Logs API"

File: content/en/docs/languages/java/instrumentation.md [hunk 3]
```diff
@@ -191,10 +191,10 @@ While the [LoggerProvider](../api-components/#loggerprovider) /
 equivalent [trace](../api-components/#tracerprovider) and
 [metric](../api-components/#meterprovider) APIs, they serve a different use
 case. As of now, `LoggerProvider` / `Logger` and associated classes represent
-the [Log Bridge API](/docs/specs/otel/logs/bridge-api/), which exists to write
-log appenders to bridge logs recorded through other log APIs / frameworks into
-OpenTelemetry. They are not intended for end user use as a replacement for Log4j
-/ SLF4J / Logback / etc.
+the [Log API](/docs/specs/otel/logs/api/), which exists to write log appenders
+to bridge logs recorded through other log APIs / frameworks into OpenTelemetry.
```
Comment (trask) [inline content/en/docs/languages/java/instrumentation.md:195]: the new (experimental) Logs API has a broader scope than existing to write log appenders

File: content/en/docs/languages/java/api-components.md [hunk 4]
```diff
@@ -4,11 +4,11 @@ weight: 11
...
   While the `LoggerProvider` / `Logger` APIs are structurally similar to the
   equivalent trace and metric APIs, they serve a different use case. As of now,
-  `LoggerProvider` / `Logger` and associated classes represent the [Log Bridge
-  API](/docs/specs/otel/logs/bridge-api/), which exists to write log appenders
-  to bridge logs recorded through other log APIs / frameworks into
-  OpenTelemetry. They are not intended for end user use as a replacement for
-  Log4j / SLF4J / Logback / etc.
+  `LoggerProvider` / `Logger` and associated classes represent the [Log
+  API](/docs/specs/otel/logs/api/), which exists to write log appenders to
+  bridge logs recorded through other log APIs / frameworks into OpenTelemetry.
```
Comment (trask) [inline content/en/docs/languages/java/api-components.md:9]: the new (experimental) Logs API has a broader scope than existing to write log appenders

File: content/en/docs/specs/status.md (hunk 1 above)
Comment (chalin) [reply content/en/docs/specs/status.md:95]: Ok, I'll revert the changes I made to once again refer to "Bridge API".

Comment (chalin) [reply content/en/docs/specs/status.md]: Reverted.

File: content/en/docs/languages/java/instrumentation.md (hunk 3 above)
Comment (chalin) [reply content/en/docs/languages/java/instrumentation.md]: Change reverted.

File: content/en/docs/languages/java/api-components.md (hunk 4 above)
Comment (chalin) [reply content/en/docs/languages/java/api-components.md:9]: Change reverted.

**Comment by chalin** [comment]
I've reverted the changes as you suggested. PTAL @trask.

**Review by trask — APPROVED** [review]
thanks @chalin 

yeah, we struggled with how to evolve the Logs Bridge API into the (experimental) Logs API in the spec repo (while still leaving alone the stable Logs Bridge API)

File: content/en/docs/zero-code/python/logs-example.md (hunk 2 above)
Comment (chalin) [reply content/en/docs/zero-code/python/logs-example.md]: Also reverted.

**Review by chalin — APPROVED** [review]

**Comment by chalin** [comment]
Finally, all checks are passing ✨

**Pull request merged**